README.md

```diff
@@ -231,7 +231,48 @@ You can find a more complex example in https://github.com/RemotePixel/remotepixe
 
 
 ## Create a Lambda layer
-`TODO`
+For this example, I will use a pyhton Lambda as the use case scenario.
+
+According to AWS [docs](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html), a Lambda layer must have the following format:
+
+```
+package.zip
+├── bin     # executables
+├── lib     # libraries
+├── lib64   # libraries 64-bit
+├── python  # python packages
+└── share   # shared libraries
+```
+
+### Creating the package
+
+To create a layer, we just need to package all the thinfs we need into a `package.zip` with the format described above. To do that, we can use a slightly modified version of the preivous script:
+
+```bash
+docker run --name lambda -itd remotepixel/amazonlinux-gdal:3.0.1 /bin/bash
+# This is just an example, installing gdal bindings for python
+docker exec -it lambda bash -c 'mkdir python'
+docker exec -it lambda bash -c 'pip install gdal==3.0.1 -no-binary :all: -t python -U'
```

Comment:
This doesn't work for py3.6 or py3.7 lambda layers, they need libs to be packaged into
- `python/lib/python${PY_VERSION}/site-packages`

Reply:
> ...they also output the archived python libs into the correct path for py3.6 and py3.7 shared libs for a lambda layer (it's not `/opt/python` any more, that only works for py2.7).

Are you 100% sure about that? I am currently using a layer, created like I described, using python 3.7. According to AWS' [docs](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path), it appears that using only `python/` works.

The following part is copied form the above link:

> Python – python, python/lib/python3.7/site-packages (site directories)
> Example Pillow
> ```
> pillow.zip
> │ python/PIL
> └ python/Pillow-5.3.0.dist-info
> ```


The `package.zip` that I generated has the following format, for python packages:

```
package.zip
│ python/osgeo
└ python/other_package
 etc...
```

And I can import the packages normally in python. Like so:
```py
from osgeo import gdal
...
```

Reply:
/shrug - didn't work for me but updates in my MR script are working OK and AFAICT the full path to the python packages in a layer becomes:
- `/opt/python/lib/python3.7/site-packages/**`

Reply:
It probably does become `/opt/python/lib/python3.7/site-packages/**`. I was only pointing out that the path, inside the `package.zip`, can be `/python/your_lib`. AWS probably does some post processing after unzipping the layer package.